.github/workflows/mariadb.yml

```diff
@@ -8,7 +8,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest]
-        ruby: [2.7, "3.0", 3.1, 3.2]
+        ruby: [2.7, 3.0, 3.1, 3.2]
```

Comment:
See https://github.com/composite-primary-keys/composite_primary_keys/pull/589

Reply:
@cfis thanks for the PR reference. Updated :)